Precise instructions on how to run mirage-www socket mode

I have followed your instructions on how to build and run mirage-www on socket mode:
```
$ cd src
$ env NET=socket FS=crunch mirage configure --unix
$ make depend
$ make
$ make run
```
but this fails to run properly, as I got:
```
[...]
read: /wiki/htcaml.md
Fatal error: exception Unix.Unix_error(1, "bind", "")
Called from file "unix/tcpip_stack_socket.ml", line 101, characters 4-41
Called from file "src/core/lwt.ml", line 652, characters 66-69
Re-raised at file "src/core/lwt.ml", line 788, characters 22-23
Called from file "src/unix/lwt_main.ml", line 34, characters 8-18
Called from file "main.ml", line 109, characters 2-28
Makefile:35: recipe for target 'run' failed
make: *** [run] Error 2
```
Using `sudo ./mir-www` all works fine.

--- a/tmpl/wiki/mirage-www.md
+++ b/tmpl/wiki/mirage-www.md
@@ -48,7 +48,7 @@ using your favourite editor.
 
 ```
 $ cd src
-$ env NET=socket FS=crunch mirage configure --unix
+$ env NET=socket PORT=8080 FS=crunch mirage configure --unix
 $ make depend
 $ make
 $ make run

